cpp/src/hash/hashing.cu

```diff
@@ -642,12 +642,14 @@ std::pair<std::unique_ptr<table>, std::vector<size_type>> hash_partition(
 std::unique_ptr<column> hash(table_view const& input,
                              hash_id hash_function,
                              std::vector<uint32_t> const& initial_hash,
+                             uint32_t seed,
```

Comment:
Is the `initial_hash` not sufficient for the `seed`? Can it be made to be sufficient? I'd like to avoid having both `initial_hash` and `seed` as it is confusing.

Reply:
Yeah, it can definitely substitute. I'll include an assertion that it should be a single value in the vector.

Reply:
Oh, I see the difference. Previously `initial_hash` requires a value per column. `seed` is just a single value. Hm, maybe both are okay then.

Reply:
The code change is simple, so it's really a question of what is most intuitive to a user. I had originally confused initial hash with seed values -- which is why I split it off -- but I'm also worried about adding too many arguments to the hash function. I think an argument of a single seed value is generic enough to include though.